lib/connection/connection_config.js

```diff
@@ -57,7 +57,6 @@ const DEFAULT_PARAMS =
   'forceGCPUseDownscopedCredential',
   'disableSamlUrlCheck',
   'representNullAsStringNull',
-  'disableSamlURLCheck'
```

Comment:
Correct property name is 'disableSamlURLCheck', so the property above should be removed.